File: Python/Count_Odd_Numbers_in_an_Interval_Range.py
# 1523. Count Odd Numbers in an Interval Range

def countOdds(self, low: int, high: int) -> int:
    if (low % 2 == 1) or (high % 2 == 1):
        return (high - low) // 2 + 1
    return (high - low) // 2

File: JavaScript/count-odd-numbers-in-an-interval-range.js
// 1523. Count Odd Numbers in an Interval Range

var countOdds = function(low, high) {
    const additionalOne = (low % 2 === 1 || high % 2 === 1) ? 1: 0;
    return Math.floor((high - low) / 2) + additionalOne;
};